emoji/emojimap.json

```diff
@@ -2,38 +2,38 @@
   "version": "draft+1",
   "description": "This ordered list of emojiwords is the conventional order for the practice of conveying checksums with emoji. Each position provides the alternative for resolving the desired emoji.",
   "emojiwords": [
-    [":+1:"],
-    [":8ball:"],
-    [":airplane:"],
-    [":alien:"],
-    [":anchor:"],
-    [":angel:"],
-    [":angry:"],
-    [":ant:"],
-    [":apple:"],
-    [":art:"],
-    [":baby:"],
-    [":baby_bottle:"],
-    [":back:"],
-    [":bamboo:"],
-    [":banana:"],
-    [":barber:"],
-    [":bathtub:"],
-    [":beer:"],
-    [":bell:"],
-    [":bicyclist:"],
-    [":bird:"],
-    [":birthday:"],
-    [":blossom:"],
-    [":blue_car:"],
-    [":boar:"],
-    [":bomb:"],
-    [":boom:"],
-    [":bow:"],
-    [":boy:"],
-    [":broken_heart:"],
-    [":bulb:"],
-    [":bus:"],
+    [":+1:", "U+1F44D"],
+    [":8ball:", "U+1F3B1"],
+    [":airplane:", "U+2708"],
+    [":alien:", "U+1F47D"],
+    [":anchor:", "U+2693"],
+    [":angel:", "U+1F607"],
+    [":angry:", "U+1F621"],
+    [":ant:", "U+1F41C"],
+    [":apple:", "U+1F34E"],
+    [":art:", "U+1F3A8"],
+    [":baby:", "U+1F476"],
+    [":baby_bottle:", "U+1F37C"],
+    [":back:", "U+1F519"],
+    [":bamboo:", "U+1F38D"],
+    [":banana:", "U+1F34C"],
+    [":barber:", "U+1F488"],
+    [":bathtub:", "U+1F6C1"],
+    [":beer:", "U+1F37A"],
+    [":bell:", "U+1F514"],
+    [":bicyclist:", "U+1F6B4"],
+    [":bird:", "U+1F426"],
+    [":birthday:", "U+1F382"],
+    [":blossom:", "U+1F33C"],
+    [":blue_car:", "U+1F693"],
```

Comment:
Unicode does not have cars by color, so I thought Police Car was the closest choice.